cluster-soak/src/main/scala/akka/kubernetes/soak/SoakMain.scala

```diff
@@ -7,24 +7,43 @@ import akka.management.cluster.bootstrap.ClusterBootstrap
 import akka.management.scaladsl.AkkaManagement
 import akka.remote.RARP
 import akka.stream.ActorMaterializer
+import com.sun.management.OperatingSystemMXBean
+import scala.concurrent.duration._
+import akka.util.PrettyDuration._
 
 object SoakMain extends App {
 
   implicit val system = ActorSystem()
   implicit val mat = ActorMaterializer()
-
   val log = system.log
+
+  import java.lang.management.ManagementFactory
+  log.info("Java version: {}", sys.props("java.version"))
+  log.info("Cores: " + Runtime.getRuntime.availableProcessors)
+  log.info("Total Memory: " + Runtime.getRuntime.totalMemory / 1000000 + "Mb")
+  log.info("Max Memory: " + Runtime.getRuntime.maxMemory / 1000000 + "Mb")
+  log.info("Free Memory: " + Runtime.getRuntime.freeMemory / 1000000 + "Mb")
+
+  val memorySize =
+    ManagementFactory.getOperatingSystemMXBean.asInstanceOf[OperatingSystemMXBean].getTotalPhysicalMemorySize
+  log.info("RAM: " + memorySize / 1000000 + "Mb")
+
+  log.info("JAVA env vars: {}", sys.env.filterKeys(_.contains("JAVA")))
+  log.info("JVM env vars: {}", sys.env.filterKeys(_.contains("JVM")))
```

Comment:
nice :+1:

Reply:
Various base images seems to set these to what they think you want so this was useful when switching between openjdk and adopt base images